patch-diff-report-tool/src/main/java/com/github/checkstyle/site/XrefGenerator.java

```diff
@@ -115,19 +128,29 @@ public final String generateXref(String name) throws IOException {
      *
      * @param name
      *        java source file path.
+     * @param shortFilePaths
+     *           {@code true} if only short file names should be used with no path.
      * @return full path to the destination of XREF file.
      */
-    private Path getDestinationPath(String name) {
-        final String newName = name + ".html";
-        final Path sourcePath = Paths.get(newName);
+    private Path getDestinationPath(String name, boolean shortFilePaths) {
         final Path destPath;
-        if (relativizationPath == null) {
-            destPath = destinationPath
-            .resolve(sourcePath.subpath(0, sourcePath.getNameCount()));
+
+        if (shortFilePaths) {
+            simpleFileNameCounter++;
+            destPath = Paths
+                    .get(destinationPath + "/File" + simpleFileNameCounter + FILE_EXTENSION);
         }
         else {
-            destPath = destinationPath
-            .resolve(relativizationPath.relativize(sourcePath));
+            final String newName = name + FILE_EXTENSION;
+            final Path sourcePath = Paths.get(newName);
+            if (relativizationPath == null) {
+                destPath = destinationPath
+                .resolve(sourcePath.subpath(0, sourcePath.getNameCount()));
```

Comment:
@romani We have checkstyle running on this code. You believe checkstyle should have put a violation here? It so, this looks like it may be a bug with `IndentationCheck` unless I am missing something else.